openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -163,7 +163,7 @@ <h3 class="section-title">${ section['display_name'] }</h3>
                                                     ${ vertical['display_name'] }
                                                 </span>
                                             </div>
-                        % if show_visual_progress and vertical.get('complete'):
```

Comment:
`show_visual_progress` what is this variable? and what this checks?

Reply:
@noraiz-anwar Can you build a sandbox for testing?

Reply:
`show_visual_progress` was there in view when outline progress feature was behind `visual_progress_enabled` waffle switch. That switch is removed now so we need to remove any associated conditioning.

Reply:
Sure, I will update this PR with snadbox link.

Reply:
Did you mean `show_visual_progress` is not being passed in context anymore?

Reply:
Correct.